src/parser/classes/livechat/items/LiveChatSponsorshipsHeader.ts

```diff
@@ -0,0 +1,39 @@
+import { Parser } from '../../../index.js';
+import { YTNode } from '../../../helpers.js';
+import type { ObservedArray } from '../../../helpers.js';
+import type { RawNode } from '../../../index.js';
+import NavigationEndpoint from '../../NavigationEndpoint.js';
+import Text from '../../misc/Text.js';
+import Thumbnail from '../../misc/Thumbnail.js';
+import LiveChatAuthorBadge from '../../LiveChatAuthorBadge.js';
+
+export default class LiveChatSponsorshipsHeader extends YTNode {
+  static type = 'LiveChatSponsorshipsHeader';
+
+  author_name: Text;
+  author_photo: Thumbnail[];
+  primary_text: Text;
+  author_badges: ObservedArray<LiveChatAuthorBadge> | null;
```

Comment:
Could you please replace the three `author_*` properties with a single `author` one that contains an instance of the `Author` class to match other nodes in the parser?

Reply:
I'm not sure whether `author` field fits better in `LiveChatSponsorshipsGiftPurchaseAnnouncement` or `LiveChatSponsorshipsHeader`.

I ended up placing `author` in `LiveChatSponsorshipsGiftPurchaseAnnouncement`,
because I think passing down the `authorExternalChannelId` data to `LiveChatSponsorshipsHeader` might be tricky to implement.